changelog/29245.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+secrets/pki: Add parameters permitted_email_addresses, permitted_ip_ranges, permitted_uri_domains, excluded_dns_domains, excluded_email_addresses, excluded_ip_ranges, and excluded_uri_domains so that the name constraints extension can be fully specified when creating root and intermediate CA certificates.
```

Comment:
Can we reword this a bit so that it calls out the omission previously?  Like "Complete the set of name constraint parameters by adding..." rest is good.

Reply:
Pushed a new version.